contracts/aFloorMigration.sol

```diff
@@ -0,0 +1,66 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity 0.7.5;
+
+import "./interfaces/IERC20.sol";
+import "./libraries/SafeMath.sol";
+import "./types/Ownable.sol";
+
+contract AlphaFloorMigration is Ownable {
+    using SafeMath for uint256;
+
+    uint256 public swapEndBlock;
+
+    IERC20 public FLOOR;
+    IERC20 public aFLOOR;
+    
+    bool public isInitialized;
+
+    modifier onlyInitialized() {
+        require(isInitialized, "not initialized");
+        _;
+    }
+
+    modifier notInitialized() {
+        require(!isInitialized, "already initialized" );
+        _;
+    }
+
+    constructor() {}
```

Comment:
Should be removed

Reply:
Snippet is now outdated; We can no longer remove this constructor as it initialises the Floor Authority logic.

```
    constructor(
        address _authority
    ) FloorAccessControlled(IFloorAuthority(_authority)) {}
```